src/compiler/diagnosticMessages.json

```
@@ -5717,6 +5717,10 @@
         "category": "Message",
         "code": 95121
     },
+    "Change all '{0}' to '{1}'": {

```

Not sure about the phrasing here - maybe

```suggestion
Switch each misplaced {0} to {1}
```

---

@sandersn

---

'misplaced' implies that `=` has a correct place in an object literal. If I could come up with a better adjective, I'd prefer "Change each X {0} to {1}.". But honestly "Change all {0} to {1}." was already fine -- a big 

## '=' &Rarr; ':'

is really all this message needs -- words don't contribute much.

---

Do I need to revert the message ‘"Change all {0} to {1}."’?

---

Would you feel better about misplaced -> misused?

---

Yeah, and if you feel that more words are better here, I don't feel *strongly* that the old wording was better.

---

@DanielRosenwasser @sandersn I've changed the message to `Switch each misused '{0}' to '{1}'`. Is it an acceptable message?

---

It's OK with me